pkg/scheduler/scheduler.go

```diff
@@ -328,17 +329,28 @@ func (sched *Scheduler) recordSchedulingFailure(fwk framework.Framework, podInfo
 	}
 
 	pod := podInfo.Pod
-	fwk.EventRecorder().Eventf(pod, nil, v1.EventTypeWarning, "FailedScheduling", "Scheduling", err.Error())
+	msg := truncateMessage(err.Error())
+	fwk.EventRecorder().Eventf(pod, nil, v1.EventTypeWarning, "FailedScheduling", "Scheduling", msg)
 	if err := updatePod(sched.client, pod, &v1.PodCondition{
 		Type:    v1.PodScheduled,
 		Status:  v1.ConditionFalse,
 		Reason:  reason,
-		Message: err.Error(),
+		Message: msg,
```

Comment:
For the pod message there shouldn't be any message size limit

Reply:
https://github.com/kubernetes/api/blob/master/core/v1/types.go#L2559 (would be best to have the full message)

Reply:
@gitirabassi reverted